view/template/patient.hbs

```diff
@@ -106,5 +118,67 @@
             allDayDefault: true,
             events: surveys
         });
+
+       
+
 	});
+
+     var scatterChartData = {
+        datasets: [{
+            label: "My First dataset",
+            data: [{
+                    x: 0,
+                    y: 0
+                }, {
+                    x: 1,
+                    y: 51
+                }, {
+                    x: 2,
+                    y: 52
+                }, {
+                    x: 3,
+                    y: 53
+                }, {
+                    x: 4,
+                    y: 54
+                }
+            ], // data
+            borderColor: "rgba(220,220,220,1)",
+            backgroundColor: "rgba(220,220,220,0.2)",
+            pointBorderColor: "rgba(220,220,220,1)",
+            pointBackgroundColor: "rgba(220,220,220,1)",
+            pointBorderWidth: "1"
+        }] // datasets
+    }; //chartData
+
+     window.onload = function() {
+        var ctx = document.getElementById("scatterChart").getContext("2d");
+        window.myScatter = Chart.Scatter(ctx, {
+            data: scatterChartData,
+            options: {
+            scales: {
+            xAxes: [{
```

Comment:
@sylvr3 time series is simple.
Replace the xAxes code here.
with this code:
https://github.com/nnnick/Chart.js/blob/v2.0-dev/samples/line-time-point-data.html#L71